app/api/dao/user.py

```diff
@@ -332,8 +305,8 @@ def change_password(user_id: int, data: Dict[str, str]):
 
         """
 
-        current_password = data["current_password"]
-        new_password = data["new_password"]
+        current_password = data.get("current_password", None)
+        new_password = data.get("new_password", None)
```

Comment:
None can be removed here as it is the default value returned
```suggestion
        current_password = data.get("current_password")
        new_password = data.get("new_password")
```